numerics/unbounded_arrays.hpp

```diff
@@ -237,7 +303,7 @@ class UnboundedUpperTriangularMatrix final {
   friend bool operator!=(UnboundedUpperTriangularMatrix const& left,
                          UnboundedUpperTriangularMatrix const& right) = default;
 
-  // For  0 ≤ i ≤ j < columns, the entry a_ij is accessed as |a(i, j)|.
+  // For  0 ≤ i < j < columns, the entry a_ij is accessed as |a(i, j)|.
```

Comment:
Why is a(i, i) not defined for a non-strictly upper triangular matrix?

Reply:
Done.